mmv1/products/pubsub/Subscription.yaml

```diff
@@ -72,13 +72,17 @@ examples:
       subscription_name: 'example-subscription'
       dataset_id: 'example_dataset'
       table_id: 'example_table'
+    test_vars_overrides:
+      policy_changed: 'acctest.BootstrapPSARoles(t, "service-", "gcp-sa-pubsub", []string{"roles/bigquery.dataEditor", "roles/bigquery.metadataViewer"})'
```

Comment:
`resource "google_project_iam_member" "viewer"` and `resource "google_project_iam_member" "editor" `should be removed from the test file with the bootstrapped iam https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/templates/terraform/examples/pubsub_subscription_push_bq.tf.tmpl#L19

Reply:
Thanks. Updated.